src/components/Login/index.jsx

```diff
@@ -1,118 +1,173 @@
+    this.setState({
+      isAuthenticating: true,
+    });
 
     if (userName === "ziv" && password === "papa") {
       this.setState(prevState => ({
-        isAuthenticated: !prevState.isAuthenticated
+        isAuthenticated: !prevState.isAuthenticated,
       }));
     } else {
       this.setState(prevState => ({
-        isShowingError: !prevState.isShowingError
+        isShowingError: !prevState.isShowingError,
+        incorrectCredentials: true,
+        isAuthenticating: false,
       }));
     }
+    // setTimeout(() => {
+    //   if (userName === "ziv" && password === "papa") {
+    //     this.setState(prevState => ({
+    //       isAuthenticated: !prevState.isAuthenticated,
+    //     }));
+    //   } else {
+    //     this.setState(prevState => ({
+    //       isShowingError: !prevState.isShowingError,
+    //       incorrectCredentials: true,
+    //       isAuthenticating: false,
+    //     }));
+    //   }
+    // }, 1000);
   }
 
   render() {
-    const { isAuthenticated, isShowingError } = this.state;
+    const {
+      isAuthenticating,
+      isAuthenticated,
+      hasNetworkError,
+      incorrectCredentials,
+    } = this.state;
     const redirectLink = "/bounce_rules";
 
     return isAuthenticated ? (
       <Redirect to={redirectLink} />
     ) : (
-      <div
-        className="row"
-        style={{
-          display: "flex",
-          justifyContent: "center",
-          alignItems: "center",
-          height: "100vh"
-        }}
-      >
-        <div
-          style={{
-            display: "flex",
-            justifyContent: "center"
-          }}
-        >
-          <div
-            className="card is-centered"
-            style={{
-              width: "30rem",
-              height: "30rem",
-              display: "flex",
-              flexDirection: "column",
-              justifyContent: "center",
-              alignItems: "center"
-            }}
-          >
-            <div
-              className="login-title"
-              style={{
-                fontWeight: "bold"
-              }}
-            >
-              SendGrid
-            </div>
-            {isShowingError && (
-              <div style={{ color: "#b71c1c", marginTop: "1rem" }}>
-                Incorrect Credentials
-              </div>
-            )}
-            <div
-              className="login-form"
-              style={{
-                marginBottom: "2rem",
-                width: "20rem"
-              }}
-            >
-              <div className="input-text-wrap">
-                <input
-                  type="text"
-                  placeholder="Username"
-                  onChange={e => this.updateField(e, "userName")}
+      <div className="login-container">
+        <Row>
+          <Column width={1} />
+          <Column width={10}>
+            {/* <div className="col-10 col-offset-1"></div> */}
+            {/* <Column width={10} offset={1}/> */}
```

Comment:
Sent you in the thread a Row and Column workaround component you can define in your project and import instead rather than the ui-components one that is messed up with offsets